scripts/docker/zeppelin-quick-start/docker-compose-zeppelin-only.yml

```diff
@@ -0,0 +1,28 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+version: "3.8"
```

Comment:
I tested this patch locally and found

> WARN[0000] /Users/chengpan/Projects/apache-zeppelin/scripts/docker/zeppelin-quick-start/docker-compose-zeppelin-only.yml: `version` is obsolete

according to https://forums.docker.com/t/docker-compose-yml-version-is-obsolete/141313 let's delete this line.

Reply:
@pan3793 I have completed this task.